### PR TITLE
New tutorial for sending custom emails

### PR DESCRIPTION
**Pull request by chandru-199**
## Purpose of this pull request

This pull request (PR) - Creating new tutorial for sending custom emails

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.4/ext-best-practices/tutorials/send-custom-email.html

### CODE REVIEW

File: src/guides/v2.3/ext-best-practices/tutorials/send-custom-email.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+---
+group: extension-best-practices
+title: Sending Custom Emails
+contributor_name: Ziffity
+contributor_link: https://www.Ziffity.com/
+---
+
+This topic describes how you can send custom emails in Magento. The below example shows how to send custom emails in Magento.
```
Comment (BarnyShergold) [inline src/guides/v2.3/ext-best-practices/tutorials/send-custom-email.md:8]: Whilst this is a great code fragment for sending emails there is no explanation to the code block, therefore it is not strictly a tutorial. The block and its requirements should be explained. What is this code and why is it there?

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Please add explanation and guide

**Comment by dobooth** [comment]
Thanks, @chandru-199. This topic could use some more explanation. I don't think we want to make a new topic that is mostly a code sample. Let us know if you have questions.

**Comment by BarnyShergold** [comment]
@chandru-199 - It's been over a week since further explanation was required for this PR. Can you please let us know if you are still working on this and a likely time to completion? If we don't hear from you within the next few days, we will have to close this PR.

**Comment by chandru-199** [comment]
Hi @BarnyShergold @dobooth.
Sorry I was not available for past few days. I will make those changes.

**Comment by dobooth** [comment]
Hi @chandru-199 Please reopen this when you are ready to proceed.

**Pull request closed without merging**